### ISSUE TITLE
Using getDisplayName function to get dayOfWeek name may not a good idea

### ISSUE THREAD

**Issue by equationl** [vanpra/compose-material-dialogs#147]
By using getDisplayName to get dayOfWeek name in Chinese will get like this:

![微信图片_20220517171323](https://user-images.githubusercontent.com/6501123/168776740-1d6cc63e-2b0d-4c7c-819c-197da7d35431.jpg)

It's will show `星` all day, which in Chinese mean nothing, such as always show `day` in English.

I know this getDisplayName is from jvm:

![image](https://user-images.githubusercontent.com/6501123/168777438-26c93df8-6bd7-491f-bd91-884bf5e88648.png)

So I test from Avd android 7.0 - Android 12. Both like this, I also using my Mi 10u(Android 12) test it, of course I get same result.

I found a same issue in other project:  [incorrect result of dayOfWeek.getDisplayName(TextStyle.NARROW, Locale.getDefault())](https://github.com/ThreeTen/threetenbp/issues/97)

Using `TextStyle.SHORT` instead `TextStyle.NARROW` will show right text.

**Comment by wojta** [comment]
I'm thinking about ability to inject the functionality from outside if necessary. Default function would be `getDisplayName()` but it can be overriden by the code similar to the one [here](https://github.com/ThreeTen/threetenbp/issues/97).  It's obviously a bug in the Android implementation, so it shouldn't be fixed by this library.  @vanpra wdyt?

**Comment by equationl** [comment]
> I'm thinking about ability to inject the functionality from outside if necessary. Default function would be `getDisplayName()` but it can be overriden by the code similar to the one [here](https://github.com/ThreeTen/threetenbp/issues/97). It's obviously a bug in the Android implementation, so it shouldn't be fixed by this library. @vanpra wdyt?

Yes. Although It not this library's problem, but may provide a function to set this paramter by user is  a good idea.

**Comment by VladZawinski** [comment]
What is the status of this issue?
Already fixed?

**Comment by equationl** [comment]
> What is the status of this issue? Already fixed?

I don't think so, But now compose already support Date Picker Dialog, You can try it.
